stock_picking_invoicing/models/stock.py

```diff
@@ -0,0 +1,458 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2013-15 Agile Business Group sagl (<http://www.agilebg.com>)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import logging
+from openerp import _
+from openerp import api
+from openerp import fields
+from openerp import models
+from openerp.exceptions import Warning
+
+_logger = logging.getLogger(__name__)
+
+INVOICE_STATE = [
+    ("invoiced", "Invoiced"),
+    ("2binvoiced", "To Be Invoiced"),
+    ("none", "Not Applicable")
+    ]
+
+
+class StockLocationPath(models.Model):
+    _inherit = "stock.location.path"
+    
+    invoice_state = fields.Selection(INVOICE_STATE, 
+                                     string="Invoice Status", default="none")
+    
+    
+    @api.v7
```

Comment:
I understand this is a forward port from openerp code but shouldnot we take the opportunity to migrate to new API?